README.md

```diff
@@ -56,28 +56,25 @@ To install all the requirements you can run:
 pip3 install -r requirements.txt
 ```
 
-### Docker container
-
-#### How to use
-
-Build the image _dmibot_ with docker:
-
-```
-$ docker build ./ -t dmibot --build-arg TOKEN=<token_API>
+## Docker
+
+### docker-compose(recommended)
+
+```json
+version: '2'
+services:
+    exambox:
+        build: .
+        container_name: dmibot
+        volumes:
+            - </path/to/settings.yaml>:/dmibot/config/settings.yaml
+            - </path/to/DMI_DB.db>:/dmibot/data/DMI_DB.db
 ```
 
-_(If you don't have a token, message Telegram's [@BotFather](http://telegram.me/Botfather) to create a bot and get a token for it)_
-
-Run the container _dmibot_:
-
-```
-$ docker run -it dmibot
-```
-
-The container **automatically** start the bot with:
-
-```
-$ cd /usr/local/dmibot && python3 main.py
+### docker cli
+```bash
+$ docker build -t dmibot .
+$ docker run -v </path/to/settings.yaml>:/dmibot/config/settings.yaml -v </path/to/DMI_DB.db>:/dmibot/data/DMI_DB.db -t dmibot
```

Comment:
I agree using the docker-compose, may we could even use `docker-compose up` command?
if we are using the volumes we should not need this `docker run -v....` command.

Moreover, if a "new user" come here and read "docker-cli" he will just run both command randomly, in the previous commit we have a better explanation of these 2 commands.
This is a learning project, so I prefer a description for the commands not a copy/paste and run.

Reply:
I think that you should know what docker is before using it.

Reply:
What you mean with 'if we are using the volumes we should not need this docker run -v.... command'?